src/backend/aspen/database/models/phylo_tree.py

```diff
@@ -61,6 +62,11 @@ class PhyloTree(Entity):
     s3_key = Column(String, nullable=False)
     name = Column(String, nullable=True)
 
+    pathogen_id = Column(
+        Integer, ForeignKey(Pathogen.id)
+    )  # TODO: change to nullable=False once we update workflows
+    pathogen = relationship(Pathogen, back_populates="phylo_trees")
```

Comment:
most of the relationships we set up in SA rely on a `foreign_keys` parameter. Do we not need those here?

Reply:
i thought we would only need that if we had multiple foreign keys to the Pathogen table